src/test/rustdoc/issue-55364.rs

```diff
@@ -0,0 +1,50 @@
+// @has issue_55364/index.html
+// @has - '//a[@class="mod"]' 'subone'
+// @has issue_55364/subone/index.html
+// These foo/bar links should refer inside `subone`
+// @has - '//div[@class="docblock"]//a/@href' 'subone/fn.foo.html'
```

Comment:
To check an attribute, we use: `//a[@href="...]`. However, I wonder if the current check is working...

Reply:
I've managed to change this to look like:

```suggestion
// @has - '//div[@class="docblock"]//a[@href="../../issue_55364/subone/fn.foo.html"]' 'foo'
```

Is that the sort of thing you were hoping for?